network/notifee.go

```diff
@@ -92,7 +93,8 @@ func (s *NotifeeService) Connected(_ lp2pnetwork.Network, conn lp2pnetwork.Conn)
 	pid := conn.RemotePeer()
 	s.logger.Info("connected to peer", "pid", pid, "direction", conn.Stat().Direction, "addr", conn.RemoteMultiaddr())
 
-	s.peerMgr.AddPeer(pid, conn.RemoteMultiaddr(), conn.Stat().Direction)
+	ai, _ := lp2ppeer.AddrInfoFromP2pAddr(conn.RemoteMultiaddr())
+	s.peerMgr.AddPeer(pid, *ai, conn.Stat().Direction)
```

Comment:
I think we should not add any peer here, indeed let's add when the `EvtPeerIdentificationCompleted` raised.
Then `PeerMgr` can decided, based on the protocols to save the peer info

Reply:
Consider some Bot or junk node may connected to our node